packages/mdc-textfield/outline/mdc-text-field-outline.scss

```diff
@@ -33,6 +33,10 @@
 }
 
 .mdc-text-field__outline {
+  @include mdc-ripple-surface;
+  @include mdc-ripple-radius;
+  @include mdc-states-base-color(text-primary-on-light);
+  @include mdc-states-press-opacity(0.12);
```

Comment:
We should reference `map-get($mdc-ripple-dark-ink-opacities, "press")` (which is .16, not .12).

Also, we should probably include styles for dark mode passing `text-primary-on-dark` and `map-get($mdc-ripple-light-ink-opacities, "press")`.

(There's no dark mode toggle under the outlined text field example at the moment either; should we add one?)

Reply:
Good call.

I plan on adding dark mode along with dense mode to the PR for adding leading/trailing icons.